src/javascript/events/page-view.js

```diff
@@ -26,8 +26,8 @@ const defaultPageConfig = function () {
  * Make the page tracking request.
  *
  * @param {Object} config - Configuration object. If omitted, will use the defaults.
- * @param {Function} callback - Callback function. Called when request completed.
- * @return {undefined}
+ * @param {Function=} callback - Callback function. Called when request completed.
```

Comment:
what does the `=` indicate? _yeah, I'm being being lazy and could DuckDuckEcosia it_

Reply:
optional

Reply:
optional?

Reply:
yeah, the equals sign states that the callback parameter is an optional parameter

Reply:
cool :) I've always defined the default value as `undefined` but this is much easier

Reply:
`undefined` is technically not correct because then jsdoc/typescript will read it as the argument is required but can be set to undefined. By using the equals sign we are telling jsdoc/typescript that the argument does not need to be passed in at all.

```js
/**
 * log things
 *
 * @param {string} a - string to log out first.
 * @param {string|undefined} b - string to log out second - optional.
 * @return {void}
 */
function log(a, b) {
	if (b) {
		console.log(a, b);
	} else {
		console.log(a);
	}
}

log('hello'); // JSDoc/TypeScript will say this is missing a required argument.
```


```js
/**
 * log things
 *
 * @param {string} a - string to log out first.
 * @param {string=} b - string to log out second - optional.
 * @return {void}
 */
function log(a, b) {
	if (b) {
		console.log(a, b);
	} else {
		console.log(a);
	}
}

log('hello'); // JSDoc/TypeScript will say this is fine because argument `b` is marked as optional
```